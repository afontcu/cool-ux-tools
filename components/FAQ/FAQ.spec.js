import Faq from './index.js'
import { mount } from '@vue/test-utils'

describe('Faq.vue', () => {
  it('is a component', () => {
    const wrapper = mount(Faq)
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  it('renders main node', () => {
    const wrapper = mount(Faq)
    expect(wrapper.classes()).toContain('Faq')
  })

  it('renders a snapshot', () => {
    const wrapper = mount(Faq)
    expect(wrapper.html()).toMatchSnapshot('default')
  })
})
