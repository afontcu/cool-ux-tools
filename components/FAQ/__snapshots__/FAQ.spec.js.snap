// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Faq.vue renders a snapshot: default 1`] = `

<div class="Faq">
  <div class="u-text-h1 u-text-special-font">
    FAQs
  </div>
  <h3 class="u-text-special-font u-text-h4">
    why did you do this?
  </h3>
  <p class="u-margin-bottom-large">
    Why not? There's a lot of cool resources out there and I needed a place to gather them. I had my own Google Sheet file, and I wanted
    to share it in a comfortable way. So here we are.
  </p>
  <h3 class="u-text-special-font u-text-h4">
    aren't you a frontend engineer?
  </h3>
  <p class="u-margin-bottom-large">
    Yes I am! But lines are blurry and sometimes I need to step into the realm of UX, for which I feel myself more insufficient.
    This is why I explored a lot of tools, to understand what were trying to solve and why.
    <br>
    <br>
    btw, I like how
    <a href="https://asinthecity.com/2011/11/10/the-difference-between-a-ux-designer-and-ui-developer/"
       title="Differences between UX designer and UI developer"
       class="FancyLink"
       target="_blank"
       rel="noopener"
    >
      this picture from Ben Melbourne
    </a>
    (the second one, obviously) articulates the difference between UX, design, frontend and so on. And I like
    doing the right things right, so research and design is critical. And "UI Developer" sounds cool, so ¯\\_(ツ)_/¯
  </p>
  <h3 class="u-text-special-font u-text-h4">
    my tool/resource/whatever isn't listed and it's cooler than you
  </h3>
  <p class="u-margin-bottom-large">
    Yeap, that's possible. Sorry to hear. I listed the tools I knew and I liked, so if yours isn't there, it's probably because
    (1) I didn't know about it or (2) other tools were cooler than yours. But I'm almost always wrong, so it must be reason #1.
    Please let me know on
    <a href="https://twitter.com/afontcu_"
       title="My Twitter"
       class="FancyLink"
       target="_blank"
       rel="noopener"
    >
      Twitter
    </a>
    or
    <a href="https://github.com/afontcu/cool-ux-tools"
       title="Github repository"
       class="FancyLink"
       target="_blank"
       rel="noopener"
    >
      Github
    </a>
    .
  </p>
</div>

`;
